beater/onboarding.go

```diff
@@ -0,0 +1,28 @@
+package beater
+
+import (
+	"time"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+func notifyListening(config Config, reporter reporter) {
+
+	var isServerUp = func() bool {
+		secure := config.SSL.isEnabled()
+		return isServerUp(secure, config.Host, 10, time.Second)
+	}
+
+	if isServerUp() {
+		logp.Info("Publishing onboarding document")
+
+		event := beat.Event{
+			Timestamp: time.Now(),
+			Fields:    common.MapStr{"listening": config.Host},
```

Comment:
hey, why was this field added?

Reply:
@jalvz See PR message. Sorry, it was not there before, only in the commit message. Forgot to update the PR message too.

Reply:
@formgeist is relevant to show in the on-boarding page in which port is the APM Server running?

Reply:
@jalvz I don't think so, we'll just check that it's there and move on. Won't be printing the port to the user in the UI. Unless you feel that might be a good thing to do?

Reply:
Ok, if not used for on-boarding is better to remove it, this is more maintenance work potentially duplicated in beats monitoring. Are you ok with removing it @ruflin ?

Reply:
The reason I put this in as I think it will help us and the users to debug potential problems with the apm-server. So in case a user does not have access to apm-server and see it running, but his agent can't send data to it, we can ask him to check this doc about the host config that is set. I expect this to be one of the most common getting started errors to happen that the wrong host / port is set. I would keep it in as it does not add any complexity as far as I can see but brings potential value. I don't think we need it in the UI for now as the doc is always accessible through discovery.